providers/gcp/connection/platform.go

```diff
@@ -111,6 +111,12 @@ func getTitleForPlatformName(name string) string {
 		return "GCP Storage Bucket"
 	case "gcp-bigquery-dataset":
 		return "GCP BigQuery Dataset"
+	case "gcp-sql-instance":
```

Comment:
Can we expand this into a platform per SQL type in GCP please

Reply:
Will do. Do you also want to have different discover flags? e.g. `--discover clou-sql-mysql` ...

Reply:
done